visualisations/line_chart/examples/line_chart_example.py

```diff
@@ -10,8 +10,15 @@
 
 line2 = [16, 5, 11, 9]
 
-lines = pd.DataFrame({'line1': line1, 'line2': line2})
+lines = pd.DataFrame({
+    'line 1': line1,
+    'line 2': line2,
+    'line 3': line2,
+    'lin;;; __': line2,
```

Comment:
This seems a bit confusing, but there might be a meaning to it?

Reply:
Name is now 'line 4' :)